### PR TITLE
Integrated Profile Page

### PR DESCRIPTION
**Pull request by dhanzy**
### What this PR does (required):
- All active contest and completed contest displays on contest page

### Screenshots / Videos (required):
- 
Screenshot (27)

### Any information needed to test this feature (required):
- Sign in

### CODE REVIEW

File: client/src/components/ProfileTabs/CardPanel/CardPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  imageFiles: string[];
+  id: string;
+}
+
+function CardPanel({ id, imageFiles, title, description, prizeAmount }: ContestProps): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Card elevation={0}>
+      <CardActionArea component={Link} to={'/contest/' + id}>
```
Comment (rajivtitus) [inline client/src/components/ProfileTabs/CardPanel/CardPanel.tsx:22]: 👍🏽

File: client/src/components/ProfileTabs/ProfileTabs.tsx [hunk 2]
```diff
@@ -23,63 +23,69 @@ function ContestTabPanel(props: TabPanelProps) {
...
-      </ContestTabPanel>
+      <Paper elevation={1}>
+        <Box p={3}>
+          <ContestTabPanel index={0} value={value}>
+            <Grid container spacing={5}>
+              {allContests
+                .filter((contest) => {
+                  return new Date(contest.deadlineDate) > new Date();
+                })
+                .map((contest, key) => (
```
Comment (rajivtitus) [inline client/src/components/ProfileTabs/ProfileTabs.tsx]: Same as the comment in the previous PR, please look into using unique values as keys as opposed to indexes.

File: client/src/components/ProfileTabs/ProfileTabs.tsx [hunk 3]
```diff
@@ -23,63 +23,69 @@ function ContestTabPanel(props: TabPanelProps) {
...
+                  </Grid>
+                ))}
+            </Grid>
+          </ContestTabPanel>
+          <ContestTabPanel index={1} value={value}>
+            {allContests
+              .filter((context) => {
+                return new Date(context.deadlineDate) <= new Date();
+              })
+              .map((contest, key) => (
```
Comment (rajivtitus) [inline client/src/components/ProfileTabs/ProfileTabs.tsx]: Same as comment above

File: client/src/pages/Dashboard/Dashboard.tsx [hunk 4]
```diff
@@ -8,7 +8,7 @@ import { useEffect } from 'react';
 import Navbar from '../../components/Navbar/Navbar';
 import { Box, Typography, Button } from '@material-ui/core';
 import AvatarDisplay from '../../components/AvatarDisplay/AvatarDisplay';
-import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
+// import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
```
Comment (rajivtitus) [inline client/src/pages/Dashboard/Dashboard.tsx]: If we are not going to be using this component, I guess we can delete the comment :)

**Review by rajivtitus — APPROVED** [review]
LGTM! Thanks for fixing the UI.
Just a few minor comments, please have a look and address them before merging.

**Comment by dhanzy** [comment]
Thanks @rajivtitus ill be working on it now

**Comment by dhanzy** [comment]
Resolved all necessary comments. @rajivtitus

File: client/src/components/ProfileTabs/ProfileTabs.tsx [hunk 5]
```diff
@@ -54,8 +54,8 @@ export default function ProfileTabs(): JSX.Element {
                 .filter((contest) => {
                   return new Date(contest.deadlineDate) > new Date();
                 })
-                .map((contest, key) => (
-                  <Grid item md={12} key={key}>
+                .map((contest) => (
+                  <Grid item md={12} key={contest.id}>
```
Comment (dhanzy) [inline client/src/components/ProfileTabs/ProfileTabs.tsx:58]: Made use of the contest id as key

**Comment by HAZberi** [comment]
LGTM. Thank you Daniel. I am merging it.

**Review by HAZberi — APPROVED** [review]

**Pull request merged**